devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -27,6 +27,9 @@ metadata {
 		capability "Switch Level"
 
 		command "pause"
+        
+ 		// NodOn
+ 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0021", manufacturer: "NodOn", model: "SIN-4-RS-20", deviceJoinName: "NodOn Roller Shutter" 
```

Comment:
Please, could Your change `deviceJoinName` to "NodOn Window Treatment" ?

Reply:
hi @KKlimczukS , I'm sorry but we prefer that the name that users see when pairing the product is close to the one they bought so that no confusion takes place. can we leave it like this, please?

Reply:
@AltyorFig Since other devices/fingerprints in this DTH follow those naming rules, please change it as requested.

Reply:
@KKlimczukS ,  I did, however in my workspace, I had selected Others instead of window processing, do you have it or should I delete my request and make another one?

Reply:
@KKlimczukS , I have another question that does not concern this code, you recently validated a multifunction module for me that worked well with my developer account and since its certification, no user can find the devices. Do you know which service can I contact?

Reply:
> @KKlimczukS , I did, however in my workspace, I had selected Others instead of window processing, do you have it or should I delete my request and make another one?

@AltyorFig Don't delete this request. You might create another request (after the current one is approved) to update it (other -> window processing).

Reply:
> @KKlimczukS , I have another question that does not concern this code, you recently validated a multifunction module for me that worked well with my developer account and since its certification, no user can find the devices. Do you know which service can I contact?

@AltyorFig 
- #77465 should be on the PROD env. in 1 week (~2th Feb)
- #77558 should be on the PROD env. in 2 weeks (~9th Feb)